services/datamanager/builtin/builtin.go

```diff
@@ -447,6 +453,10 @@ func (svc *builtIn) Reconfigure(
 		svc.startSyncScheduler(svc.syncIntervalMins)
 	} else {
 		svc.closeSyncer()
+		if svc.ticker != nil {
```

Comment:
Should this be done before closing the syncer? If this ticks between these two lines, would there be a risk of a NPE?